cmd/nodecmd/ssh.go

```diff
@@ -75,6 +77,11 @@ func sshNode(_ *cobra.Command, args []string) error {
 			// try to detect nodeID
 			for clusterName := range clustersConfig.Clusters {
 				clusterHosts, _ := ansible.GetInventoryFromAnsibleInventoryFile(app.GetAnsibleInventoryDirPath(clusterName))
+				monitoringInventoryPath := filepath.Join(app.GetAnsibleInventoryDirPath(clusterName), constants.MonitoringDir)
```

Comment:
why do we want to include monitor host to ssh command?
are yiou trying to allow ssh command execution for hostID for monitoring ?

Reply:
yes. I am adding that so can use ssh command also with the monitoring instance ip, cloud id, and whatever

Reply:
this command should give access to all resources